### PR TITLE
Rotation status property of block added.

### PR DESCRIPTION
**Pull request by gstankov**
Hey,

made a proposed change to the 'Block' model to include rotation status of a block.

Can you please verify? 

Br,

### CODE REVIEW

File: src/Model/Block.php [hunk 1]
```diff
@@ -106,6 +111,8 @@ public function rotate(): self
         $this->setWidth($this->getHeight());
         $this->setHeight($w);
 
+        $this->rotated = true;
```
Comment (Padam87) [inline src/Model/Block.php:114]: `$this->rotated = !$this->rotated;`

I'd just negate it. That way, if rotate is called multiple times for some reason, we still know which is the original.

File: src/Model/Block.php [hunk 2]
```diff
@@ -106,6 +111,8 @@ public function rotate(): self
...
+
         return $this;
     }
```
Comment (Padam87) [inline src/Model/Block.php:118]: A getter is also needed. `isRotated`

**Review by Padam87 — CHANGES_REQUESTED** [review]

**Comment by gstankov** [comment]
Ok, made the changes.

Also I wanted to write a test case for it but couldn't run them. Getting some errors regarding 'PHPUnit' framework.

What's the correct command to run them? Tried 'phpunit --bootstrap ./vendor/autoload.php tests' with no success.

Br,

**Comment by Padam87** [comment]
I just ran phpunit without parameters, it picked up the `phpunit.xml.dist` file as it should.

**Review by gstankov — COMMENTED** [review]
Hi Adam,

i managed to create and run a test. Can you please take a look.

Thanks in advance, br

**Pull request merged**